lib/signs_ui/config/utilities.ex

```diff
@@ -0,0 +1,28 @@
+defmodule SignsUi.Config.Utilities do
+  @moduledoc """
+  Utility functions for interacting with the arinc_to_realtime.json file
+  """
+
+  def clean_configs(sign_configs) do
+    registered_ids = get_realtime_ids()
+
+    :maps.filter(
```

Comment:
Depending on which version of elixir signs_ui is running, you might be able to use `Map.filter/2` for this

Reply:
Signs UI is currently running on Elixir 1.11 so Map.Filter isn't available. We should probably update the version though soon